tools/mason/MasonHelp.chpl

```diff
@@ -35,10 +35,10 @@ proc masonHelp() {
   writeln('Options:');
   writeln('    -h, --help          Display this message');
   writeln('    -V, --version       Print version info and exit');
-  writeln('    --list              List installed commands');
   writeln();
-  writeln('Some common mason commands are (see all commands with --list):');
+  writeln('Some common mason commands are:');
   writeln('    new         Create a new mason project');
+  writeln('    init        Initializes a library project inside a given directory or path.');
```

Comment:
Some suggested word-smithing for conciseness / consistency with other help outputs:

```suggestion
  writeln('    init        Initialize a mason project inside an existing directory');
```